styles/base.css

```diff
@@ -1,3 +1,5 @@
+/*TODO: Create unique font sizes, styles, typefaces, etc to persists across the page to add interesting aesthetics.  */
```

Comment:
I really like the look of your site- the fonts and color choices are a lot of fun. I think you could also take better advantage of the multiple breakpoints you've got going to tweak the structure, layout, and design of your elements in browser, tablet, and mobile sizes.  Right now, your site sort of looks the same whether the user is accessing via mobile or full on desktop. Try playing around with different displays (inline, or inline block maybe) so that the information is more easily accessible with minimal scrolling for browser users. You could even add a wrap to the body, so that the content stays centered in the window no matter the size of the screen.